### PR TITLE
Add a new field `store_id` in Debug::get_metrics RPC.

### PR DESCRIPTION
**Pull request by hicqu**
Add a new RPC to get store infomations from TiKV.

### CODE REVIEW

File: proto/debugpb.proto [hunk 1]
```diff
@@ -224,4 +227,11 @@ message GetRegionPropertiesRequest {
...
     repeated Property props = 1;
-}
\ No newline at end of file
+}
+
+message StoreInfoRequest {
+}
+
+message StoreInfoResponse {
+    uint64 id = 1;
```
Comment (BusyJay) [inline proto/debugpb.proto]: Do we need such standalone RPC? Can it be included in metrics response?

**Comment by breeswish** [comment]
LGTM

**Comment by BusyJay** [comment]
@LinuxGit PTAL

**Comment by LinuxGit** [comment]
LGTM

**Pull request merged**